packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -1024,6 +1024,7 @@ defineExpose({
 });
 
 onBeforeUnmount(() => {
+	valueChangedDebounced.cancel();
```

Comment:
Ideally this would be `flush` but it seems the activeNode already changed at this point, causing the change to go the wrong node, hence `cancel` instead